omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx.conf.erb

```diff
@@ -148,6 +150,13 @@ http {
             more_clear_headers Server;
             access_log /var/log/opscode/nginx/rewrite-port-80.log;
             return 301 https://$host$request_uri;
+            location @errorrespfilter {
```

Comment:
Let's include a comment here that explains why we're doing this filter.  Same in api lb conf.